docs/ja/user-guide/configuration/index.md

```diff
@@ -15,80 +15,98 @@ toc:
-        </div>
-        <div id="settings" class="hidden">
-            <h4>Settings</h4>
-            <p>Configurable settings for any additional build plugins added to Screwdriver.cd.</p>
-        </div>
+</div>
+<pre data-md-type="block_code" data-md-language=""><code><div class="yaml-side">
+    <div id="requires" class="hidden">
+        <h4>Requires</h4>
+        <p>ジョブ実行のトリガーとなる1つまたは複数のジョブの配列を設定します。"requires: ~pr"が設定されたジョブはプルリクエストイベントにより実行されます。"requires: ~commit"が設定されたジョブはプッシュイベントにより実行されます。"requires: ~sd@123:main"が設定されたジョブはパイプライン"123"の"main"ジョブの完了により実行されます。"requires: [deploy-west, deploy-east]"が設定されたジョブは"deploy-west"と"deploy-east"の両方のジョブの成功後に実行されます。"注意: ~ のジョブはOR条件で、~ なしのジョブはジョインの働きをします。"</p>
+    </div>
+    <div id="shared" class="hidden">
+        <h4>Shared</h4>
+        <p>全てのジョブの適用されるグローバル設定を定義します。Sharedの設定は各ジョブにマージされますが、それぞれのジョブで更に設定が行われている場合はその設定により上書きされます。</p>
+    </div>
+    <div id="environment" class="hidden">
+        <h4>Environment</h4>
+        <p>ビルドに必要な環境変数のキーと値の組み合わせです。ジョブに設定できる全ての設定はSharedにも設定できますが、個別のジョブの設定により上書きされます。</p>
+    </div>
+    <div id="settings" class="hidden">
+        <h4>Settings</h4>
+        <p>Screwdriver.cdに追加されたビルドプラグインの設定です。</p>
+    </div>
+    <div id="annotations" class="hidden">
+        <h4>Annotations</h4>
+        <p>アノテーションはキーバリューのペアです。パイプラインまたはジョブ単位で設定することができます。アノテーションのキーバリューは任意に設定することができ、例えば、ビルドの実行方法を変更することができます。Screwdriverのクラスタ管理者にビルドの実行方法を変更するために、どのようなアノテーションがサポートされているか確認してください。</p>
+    </div>
+    <div id="executor" class="hidden">
+        <h4>Executor annotation</h4>
+        <p>デフォルト以外のExecutorでのビルドを実行したい場合に設定します。`jenkins`, `k8s-vm`が設定可能です。</p>
+    </div>
+    <div id="cpu" class="hidden">
+        <h4>CPU annotation</h4>
+        <p>`k8s-vm` Executor を利用する場合にVMに割り当てられるCPUを設定します。`LOW`はデフォルトで設定され、2CPUが割り当てられ、`HIGH`は6CPU割り当てられます。</p>
+    </div>
+    <div id="ram" class="hidden">
+        <h4>RAM annotation</h4>
+        <p>`k8s-vm` Executor を利用する場合にVMに割り当てられるRAMを設定します。`LOW`はデフォルトで設定され、2GBのメモリが割り当てられ、`HIGH`は12GBのメモリが割り当てられます。</p>
         <div id="email" class="hidden">
             <h4>Email</h4>
-            <p>Emails addresses to send notifications to and statuses to send notifications for.</p>
+            <p>通知を送信するEmailアドレスと、通知を送信するステータスを設定します。</p>
         </div>
         <div id="jobs" class="hidden">
             <h4>Jobs</h4>
             <p>ビルドの挙動を定義するジョブのリストです。</p>
         </div>
-        <div id="main-job" class="hidden">
-            <h4>Main</h4>
-            <p>唯一の必須ジョブです。このジョブはコードに変更が行われた際に自動的に実行されます。</p>
-        </div>
         <div id="image" class="hidden">
             <h4>Image</h4>
-            <p>ビルドで利用されるDockerイメージを指定します。この例ではテンプレートを使用しており、{{NODE_VERSION}}のように波括弧で囲まれた変数が展開されます。 この変数はmatrixで指定されている値に変更され、それら複数のイメージを利用したビルドが並列に実行されます。</p>
+            <p>ビルドで利用されるDockerイメージを指定します。ここでの設定は"docker pull"コマンドを利用する場合と同じものです。</p>
         </div>
         <div id="matrix" class="hidden">
             <h4>Matrix</h4>
```

Comment:
The original guide doesn't has an explanation of `Matrix`. It might be unnecessary.

Reply:
matrix was removed since it isn't supported yet.

Reply:
removed about `Matrix`.